packages/edit-post/src/components/visual-editor/style.scss

```diff
@@ -9,7 +9,7 @@
 
 .edit-post-visual-editor .block-editor-writing-flow__click-redirect {
 	// Collapse to minimum height of 50px, to fully occupy editor bottom pad.
-	height: 50px;
+	height: 240px;
```

Comment:
Mentioned in https://github.com/WordPress/gutenberg/pull/15165#issuecomment-487095968 , I sense the value here was intentionally aligned to the `50px` in:

https://github.com/WordPress/gutenberg/blob/b9e2e893f6a0ff30545869466a6f3c6892ad502c/packages/edit-post/src/components/visual-editor/style.scss#L3

https://github.com/WordPress/gutenberg/blob/b9e2e893f6a0ff30545869466a6f3c6892ad502c/packages/edit-post/src/components/visual-editor/style.scss#L15

Where, if we were to change the value, the others ought similarly be updated.

I don't recall if or what specific difference it might make, or if even the visual editor needs _any_ bottom padding when the click redirect is present.

Reply:
I actually tried changing those 50px to -240px to match, and the result got weird. I don't fully understand what those 50px margin do, and I notably don't understand why it got weird. I suppose that's worth examining, otherwise I can record a GIF of what happens tomorrow. Heading out for the night now.

Reply:
updating the margin to correlate with the height works all right (I have no idea what the effect is). However the padding at line 3 is the padding that pushes content below the top bar. That should be left as it is @aduth